src/components/application_manager/src/commands/mobile/alert_request.cc

```diff
@@ -63,31 +63,27 @@ AlertRequest::AlertRequest(const MessageSharedPtr& message,
 AlertRequest::~AlertRequest() {}
 
 bool AlertRequest::Init() {
-  /* Timeout in milliseconds.
-     If omitted a standard value of 10000 milliseconds is used.*/
-  if ((*message_)[strings::msg_params].keyExists(strings::duration)) {
-    default_timeout_ =
-        (*message_)[strings::msg_params][strings::duration].asUInt();
-  } else {
-    const int32_t def_value = 5000;
```

Comment:
The defaul value should still be included. In case there is no timeout supplied, default_timeout_ +=def_value;

Reply:
@JackLivio Right but not here. It is included in MOBILE_API as default value. Therefore if mobile app will send request without specific duration - default value will be used. See [Alert request](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/MOBILE_API.xml#L3601)

Reply:
ok thank you